product_customer_code_invoice/README.rst

```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================================
+Product Customer Code for Account Invoice
+=========================================
+
+This module allows to use product customer code in Account Invoice (see product_customer_code module)
```

Comment:
This module allows to use product customer code (instead of the standard Internal Reference) in Account Invoice (see product_customer_code module).